lib/compiler/src/compile.erl

```diff
@@ -719,7 +719,9 @@ core_passes() ->
 
 kernel_passes() ->
     %% Optimizations that must be done after all other optimizations.
-    [{pass,sys_core_bsm},
+    [{pass,sys_core_alias},
```

Comment:
Discussion point: do we want to apply this pass after all other optimizations as well? Or can we see benefits from applying it earlier?